llvm/lib/MC/RepoObjectWriter.cpp

```diff
@@ -649,9 +650,13 @@ pstore::index::digest RepoObjectWriter::buildCompilationRecord(
   CompilationHash.update(Triple.size());
   CompilationHash.update(Triple);
 
+  // Only record all compilation memebers one time.
+  SmallSet<TicketNode *, 4> CMembers;
```

Comment:
May be change the name to something like: 'SeenCompilationMembers'.

Reply:
Changed to 'SeenCompilationMembers'.